src/App.js

```diff
@@ -71,7 +71,9 @@ class App extends React.Component {
   render() {
     const { location } = this.props;
     const activeItem = this.menu.find(
-      item => location.pathname.indexOf(item.to) > -1
+      item =>
+        (item.to === '/' && location.pathname.indexOf('/ipsum') > -1) ||
```

Comment:
Please note that this change will reflect only the default routing. I suggest to use the logic of `routes.js` instead of hardcoding the paths that has sub pages.

Reply:
ok - yea i agree w/ you, it is a bit too prescriptive for a demo. I'm definitely welcome to an alternative :). This was the shortest path for the example here I could think of a.t.m.

Reply:
@priley86, I have a better alternative here :) https://github.com/patternfly/patternfly-react-demo-app/pull/45